packages/video_player/video_player/README.md

```diff
@@ -136,3 +165,15 @@ and so on.
 To learn about playback speed limitations, see the [`setPlaybackSpeed` method documentation](https://pub.dev/documentation/video_player/latest/video_player/VideoPlayerController/setPlaybackSpeed.html).
 
 Furthermore, see the example app for an example playback speed implementation.
+
+### Closed Captioning
+
+#### Caption Files
```

Comment:
These sections are very short, and closely related; I don't think we need sub-headings for each part.